frontend/src/components/ui/data-table.ts

```diff
@@ -61,10 +61,12 @@ export class DataTable extends TailwindElement {
     }`;
     return html`
       <btrix-table
-        class="overflow-auto rounded border"
+        class="relative h-full w-full rounded border"
         style=${gridAutoColumnsStyle}
       >
-        <btrix-table-head class="rounded-t border-b bg-neutral-50">
+        <btrix-table-head
+          class="sticky top-0 z-10 rounded-t-[0.188rem] border-b bg-neutral-50"
```

Comment:
Minor thing, but `0.188rem` is neither clear about what it represents, nor is it exactly 3px. I don't think we intend on changing the root rem scale, but if we do want to be able to at some point, `0.1875rem` would also work here.
```suggestion
          class="sticky top-0 z-10 rounded-t-[3px] border-b bg-neutral-50"
```

Reply:
Changing to 1875.  I agree that we probably won't be changing the root em scale.... But it's also how we've defined everything else?  IDK why I thought this shouldn't be exactly 3px, checking again that appears to be the correct inner border size.

Is there a way of fixing this that doesn't require setting an additional inner border?  Seems like kinda a janky solution.

Reply:
Not really, border radii don't propagate inwards, and the table body isn't what's sticky, so there's no reason for its border to stay visible at the top when you scroll